subsys/mgmt/mcumgr/smp_bt.c

```diff
@@ -64,10 +114,21 @@ static ssize_t smp_bt_chr_write(struct bt_conn *conn,
 	zephyr_smp_rx_req(&smp_bt_transport, nb);
 
 	return len;
+#endif
 }
 
 static void smp_bt_ccc_changed(const struct bt_gatt_attr *attr, uint16_t value)
 {
+#ifdef CONFIG_MCUMGR_SMP_PKASM_BT
+	if (value == BT_GATT_ITER_STOP && in_progress) {
+		struct smp_bt_user_data *ud = zephyr_smp_pkasm_get_ud(&smp_bt_transport);
+
+		bt_conn_unref(ud->conn);
```

Comment:
`bt_conn_unref`, should be followed by ud->conn = NULL;

Reply:
Done